tests/shared/drive-constraints.spec.js

```diff
@@ -526,29 +717,53 @@ describe('Shared: DriveConstraints', function() {
 
       it('should return false if the drive is not large enough and is a source drive', function() {
         m.chai.expect(constraints.isDriveValid(this.drive, {
-          path: path.join(this.mountpoint, 'rpi.img'),
-          size: 5000000000
+          path: path.join(this.mountpoint, __dirname, 'rpi.img'),
```

Comment:
Ummm, huh? Shouldn't this have been left as `path: path.join(this.mountpoint, 'rpi.img'),` ?